### PR TITLE
add new paragrah(backup type) to backup-*-using-br.md

### PR DESCRIPTION
**Pull request by wxiaoke**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.3 (TiDB Operator 1.3 versions)
- [x] v1.2 (TiDB Operator 1.2 versions)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/backup-to-aws-s3-using-br.md [hunk 1]
```diff
@@ -202,6 +202,172 @@ Ad-hoc 备份支持全量备份与增量备份。Ad-hoc 备份通过创建一个
...
+        # timeAgo: ${time}
+        # checksum: true
+        # options:
+        # - --lastbackupts=420134118382108673
+      # Only needed for TiDB Operator < v1.1.10 or TiDB < v4.0.8
+      from:
+        host: ${tidb_host}
+        port: ${tidb_port}
+        user: ${tidb_user}
+        secretName: backup-demo1-tidb-secret
```
Comment (csuzhangxc) [inline zh/backup-to-aws-s3-using-br.md:237]: `from` 这几行也干脆注释掉吗？`Only needed for TiDB Operator < v1.1.10 or TiDB < v4.0.8`

Comment (wxiaoke) [reply zh/backup-to-aws-s3-using-br.md:237]: 之前的示例是这样的，直接和之前的示例保持一致了

**Comment by ran-huang** [comment]
备份示例 cp 到 v1.1 后需要对应修改吗？

**Comment by wxiaoke** [comment]
>

**Comment by ran-huang** [comment]
@wxiaoke 麻烦签一下 CLA~ https://cla-assistant.io/pingcap/docs-tidb-operator?pullRequest=1622

**Comment by wxiaoke** [comment]
> @wxiaoke 麻烦签一下 CLA~ https://cla-assistant.io/pingcap/docs-tidb-operator?pullRequest=1622

已经签过了～不知道为啥状态一致没更新过来

**Pull request closed without merging**